### PR TITLE
docker-compose for easy redis setup

### PR DESCRIPTION
**Pull request by adityabisoi**
This PR involves adding a ```docker-compose``` file to easily setup ```redis```.
It closes issue #29 
@ankitjena Please review and suggest me any changes.

### CODE REVIEW

**Comment by ankitjena** [comment]
@adityabisoi Also update readme on how to use docker

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    volumes:
+      - ../data/redis:/data
+    entrypoint: redis-server --appendonly yes
+    restart: always
+
+networks:
+  redis-net:
+
+volumes:
+  redis-data:
```
Comment (ankitjena) [inline docker-compose.yml]: We don't need these lines

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
+  redis:
+    container_name: redis
```
Comment (ankitjena) [inline docker-compose.yml]: Let's clean this up a little, we don't need a lot of things like, `container_name`, `volumes`, `entrypoint`.

**Comment by adityabisoi** [comment]
I have fixed the redundancies and added command for ```docker-compose```. Please review. Thanks.

File: README.md [hunk 3]
```diff
@@ -4,7 +4,12 @@ A book uploader bot that transfers documents from public libraries such as Googl
 ## Getting Started
 1. Install [Node](https://nodejs.org/en/download/). Check for Node and NPM correct install using `node -v` and `npm -v` on the terminal.
 2. Install [Docker toolbox](https://docs.docker.com/toolbox/toolbox_install_windows/).
-3. Start redis service in the background using : `docker run --name redis -p 6379:6379 redis` and `docker start redis` . Note that redis server can also be run in other different possible ways, apart from the one mentioned in this point.
+3. Start redis service in the background 
+* Using Docker :
+`docker run --name redis -p 6379:6379 redis` and `docker start redis` . 
+
+* Using `docker-compose` : 
+`docker-compose up`
```
Comment (ankitjena) [inline README.md]:
```suggestion
`docker-compose up -d`
```

**Comment by ankitjena** [comment]
@adityabisoi Looks good, commit the suggestion and we are good to merge.

**Comment by adityabisoi** [comment]
@ankitjena Thanks !

**Comment by vaishnavi-janardhan** [comment]
@ankitjena Please add labels `gssoc20` and either of `beginner`, `easy`, `medium` and `hard`.

**Comment by ankitjena** [comment]
@vaishnavi-janardhan Sure

**Pull request merged**